js/dataframe/static/StaticDataframe.svelte

```diff
@@ -85,11 +82,8 @@
 		{label}
 		{row_count}
 		{col_count}
-		values={value}
+		{value}
 		{headers}
-		on:change={({ detail }) => {
```

Comment:
I had to remove this `on:change` handler -- otherwise I'd get an infinite loop. I didn't see any repercussions but I don't think this is the right thing to do

Reply:
The static dataframe never updates from inside so this won't cause any issues (and I couldn't find any from testing). This is probably a leftover from a refactor or something.